library/alloc/src/slice.rs

```diff
@@ -122,16 +122,6 @@ pub use core::slice::{RSplitN, RSplitNMut, SplitN, SplitNMut};
 // Basic slice extension methods
 ////////////////////////////////////////////////////////////////////////////////
 
-// HACK(japaric) needed for the implementation of `vec!` macro during testing
-// N.B., see the `hack` module in this file for more details.
-#[cfg(test)]
-pub use hack::into_vec;
-
-// HACK(japaric) needed for the implementation of `Vec::clone` during testing
-// N.B., see the `hack` module in this file for more details.
-#[cfg(test)]
-pub use hack::to_vec;
-
 // HACK(japaric): With cfg(test) `impl [T]` is not available, these three
 // functions are actually methods that are in `impl [T]` but not in
 // `core::slice::SliceExt` - we need to supply these functions for the
```

Comment:
I think this comment is actually no longer correct, and the above exports are also as well... but I'm not completely sure.